sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/SharedThriftServer.scala

```diff
@@ -50,14 +53,21 @@ trait SharedThriftServer extends SharedSparkSession {
       hiveServer2.stop()
     } finally {
       super.afterAll()
+      SessionState.detachSession()
```

Comment:
This fix is added to `SharedThriftServer ` not `SharedThriftServer` for a lower scope, but I think most of hive and `SharedSparkSession` tests that need a dedicated JVM may cause by this.